ui/package.json

```diff
@@ -29,6 +28,7 @@
     "eject": "react-scripts eject"
   },
   "//": [
+    "vega-lib and vega-lite are peerDependencies of react-vega-lite.",
```

Comment:
Is it possible to split this into two comments? Leave 32 here, move 31 to 17.

Reply:
Yup it is, done.